docs/sources/contribute/release-notes/index.md

```diff
@@ -34,185 +34,102 @@ The Grafana Cloud Traces updates are published using the What's new.
-      - [Docs squad office hours (late)](https://docs.google.com/document/d/12XK3XYEZWU3uIPluq3sn5HSGqWHBHJkktqlyQZHj_Uw/edit#heading=h.9o53ccbx7xrw)
-
-      {{< /admonition >}}
-
    1. Work with your team to review and finalize the generated pull request.
 
-   1. Merge your PR in time for your feature release date.
+   {{< admonition type="caution" >}}
+   The Grafana Labs documentation team doesn't automatically review these pull requests; teams that create What's new entries are responsible for determining their own review process.
+   {{< /admonition >}}
 
-      Merging your PR ensures your entry is live on the date you entered and it automatically updates the status of your entry in the CMS.
+1. Publish your entry by the release date you've entered.
 
-1. To publish your entry from the CMS, follow these steps:
+   - To publish from GitHub, merge your PR.
+   - To publish from the CMS:
 
    1. In the **Status** drop-down, click **Ready**.
    1. In the **Publish** drop-down, click **Publish now**.
-      The entry appears in [What's new in Cloud](https://grafana.com/docs/grafana-cloud/whats-new/) on the date you entered.
+
+<!-- vale Grafana.GoogleWill = YES -->
+
+The entry appears in [What's new in Cloud](https://grafana.com/docs/grafana-cloud/whats-new/) on the release date you've entered. If the date is in the past, it appears immediately.
 
 For Grafana versioned releases, the content you enter in the CMS is published in the versioned What's new at a later date.
 To understand the process of creating release notes for Grafana versioned releases, refer to [Create the versioned release notes](#create-the-versioned-release-notes).
 
+{{< admonition type="caution">}}
 If you add an entry to the CMS after the relevant versioned What's new has already been published, you'll need to open a PR to also add it to the versioned What's new yourself.
+{{< /admonition >}}
 
-#### Updated release stage
+### CMS fields
 
-If you've previously created a What's new entry for a feature when it was in an early release stage and you want to announce that the feature has moved into a new release stage, create a new entry.
-For example, you published a What's new entry when the feature was in public preview and now the feature is in general availability.
+<!-- vale Grafana.GoogleWill = NO -->
+<!-- prettier-ignore-start -->
+
+| Field | Description | Guidance |
+|---|---|---|
+| FEATURE NAME | Short headline for the feature. | For example, _Grafana OnCall integration for Alerting_. |
+| FEATURE RELEASE DATE | Date and time in UTC that you want this note to be live. | This should also be the feature release date. If the feature is behind a feature toggle and gets rolled out only to a fraction of users the date is when the feature was first available to users opting in.<br></br>If you've opened a review PR, you must merge it before the date you've added here. If you enter a date that has passed, the website publishes the note on the next build. |
```

Comment:
```suggestion
| FEATURE RELEASE DATE | Date and time in UTC that you want this note to be live. | This should also be the feature release date. If the feature is behind a feature toggle and gets rolled out only to a fraction of users, the date is when the feature was first available to users opting in.<br></br>If you've opened a review PR, you must merge it before the date you've added here. If you enter a date that has passed, the website publishes the note on the next build. |
```

Reply:
We shouldn't use `<br>` tags to introduce line breaks. If we want multiple paragraphs, we should use `<p>` tags: https://developers.google.com/style/tables#multi-paragraph-table-cells

Reply:
I'll fix that in VS code and push again.